docs/source/subgraphs.md

```diff
@@ -1,82 +1,51 @@
 ---
 title: 'Working with federated graphs'
 sidebar_title: 'Federated graphs'
-description: 'How to use Rover to work with federated graphs and their implementing services'
+description: '(Graphs represented by multiple implementing services)'
 ---
 
-Rover is Apollo's preferred way of working with federated graphs, and provides
-a number of useful utilities for doing just that.
+> This article applies only to federated graphs. When working with a non-federated graph, see [Working with non-federated graphs](./graphs).
 
-> Everything on this page applies only to federated graphs and their
-> implementing services. For a guide on how to work with a non-federated graph,
-> check out [Working with Graphs](./graphs).
+## Fetching a service schema
 
-## Introspecting Implementing Services
+### Fetching from Apollo Studio
 
-Implementing services (subgraphs) in a federated graph have additional
-information that can't be expressed in a normal introspection request like
-`@key` values, `@external` definitions, etc. There is a special field available
-in subgraphs implementing the Apollo Federation spec, which can be used to get
-the full schema of a subgraph.
+> This requires first [authenticating Rover with Apollo Studio](./configuring/#authenticating-with-apollo-studio).
 
-Rover's `subgraph introspect` command makes fetching SDL from a running subgraph
-easier than ever:
+You can use Rover to fetch the current schema of any implementing service that belongs to a Studio graph and variant that Rover has access to.
+
+Run the `subgraph fetch` command, like so:
 
 ```bash
-rover subgraph introspect http://localhost:4001
+rover subgraph fetch my-graph@my-variant --service-name accounts
 ```
 
-This command outputs the "special" introspection result to `stdout`, which can
-be piped to other commands or output to a file.
-
-> For more on how `stdout` works, see [Essential Concepts](./essentials#using-stdout).
-
-> TODO: headers API
+The argument `my-graph@my-variant` in the example above specifies the ID of the Studio graph you're pushing to, along with which [variant](https://www.apollographql.com/docs/studio/org/graphs/#managing-variants) you're pushing to.
 
-## Pushing a subgraph to Apollo Studio
+> You can omit `@` and the variant name. If you do, Rover uses the default variant, named `current`.
 
-> All of the following examples require you to first create an account with
-> Apollo Studio, [authenticate Rover]
-> (./configuring#authenticating-with-apollo-studio) and create a new graph to
-> publish to.
+The `--service-name` option is also required. It must match the implementing service you're fetching the schema for.
```

Comment:
I think `--name` is fine for this command because `subgraph` indicates what you're providing the name _of_.

_Most_ ideally from my perspective we bake the subgraph name into the graph identifier as was discussed a bit on Slack yesterday, but in the absence of that, no issue with `--name`. I won't make the change in this PR, but feel free to update whenever you finalize

Reply:
Great! I think I agree with adding it to a graph ref! Whenever that work lands on the backend (assuming that gets finalized), we'll probably propogate those changes to Rover